Main.java

```diff
@@ -9,7 +10,53 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+     // create a scanner for user input
+Scanner input = new Scanner(System.in);
+
+
+// ask the casier for the price 
+ System.out.println("Please enter the amount the customer spent");
+
+// get the discount price from the user
+double price = input.nextDouble();  
+
+double total = 0;
+double save = 0;
+// variable for discount for 0-40$ 10%
+double A =  price*.10;
+
+//variable for discount for $40.01 - $80.00
+double B = price*.20;
+
+//variable for discount for $80.01 - $120.00
+double C = price*.30;
+
+//variable for discount for over $120
+double D = price*.40;
+
+ if(price> 0 && price <= 40){
+   save = price *.10;
+   total = price - save;
+ System.out.println("They will receive 10% off.");
+ } else {
+ if(price> 40.01 && price <= 80){
+ System.out.println("They will receive 20% off.");
+ } else {
+  if(price> 80.01 && price >= 120){
+  System.out.println("They will receive 30% off.");
+ } else {
+  if(price> 120.01){
+   System.out.println("They will receive 40% off.");
+  }
+ }
+ }
```

Comment:
Watch your indentation. If there is code inside of an if statement, it should be TABBED over an extra bit to show it is inside of the if-statement. Try hitting the autoformat button in the top right hand corner. Also, it looks like you have an if statement inside of another if statement. I'm not sure this is really what you want to do.